docs/src/manifolds/vector_bundle.md

```diff
@@ -23,21 +23,19 @@ It is used for example in musical isomorphisms (the [`flat`](@ref) and [`sharp`]
 
 ## Example
 
-The following code defines two points on a tangent bundle of the sphere $S^2$ and calculates distance between them, distance between their base points and norm of one of these tangent vectors.
+The following code defines a point on a tangent bundle of the sphere $S^2$ and a tangent vector to that point and calculates an approximation of the exponential in the Sasaki metric using 1000 steps.
```

Comment:
```suggestion
The following code defines a point on the tangent bundle of the sphere $S^2$ and a tangent vector to that point and calculates an approximation of the exponential in the Sasaki metric using 1000 steps.
```

Reply:
That line is also a little long, we could add some new lines in the code maybe.

Reply:
OK, I'll try to rephrase it.